src/test/java/com/code_intelligence/jazzer/mutation/mutator/StressTest.java

```diff
@@ -90,6 +88,16 @@ public static Stream<Arguments> stressTestCases() {
             exactly(
                 null, emptyList(), singletonList(null), singletonList(false), singletonList(true)),
             distinctElementsRatio(0.30)),
+        arguments(
+            new TypeHolder<@NotNull Map<@NotNull String, @NotNull String>>() {}.annotatedType(),
+            "Map<String,String>", manyDistinctElements(), manyDistinctElements()),
+        arguments(new TypeHolder<Map<@NotNull String, @NotNull String>>() {}.annotatedType(),
+            "Nullable<Map<String,String>>", manyDistinctElements(), manyDistinctElements()),
+        arguments(
+            new TypeHolder<@WithSize(
+                min = 1, max = 3) @NotNull Map<@NotNull Integer, @NotNull Integer>>() {
+            }.annotatedType(),
+            "Map<Integer,Integer>", hasMapSizeInClosedRange(1, 3), hasMapSizeInClosedRange(1, 3)),
```

Comment:
Do the different values after mutations satisfy `manyDistinctElements`?

Reply:
Yes, added it as well.